src/gui/widgets/general_settings_page.cpp

```diff
@@ -30,6 +30,7 @@
 #include <QFormLayout>
 #include <QLineEdit>
 #include <QMessageBox>
+#include <QPushButton>
```

Comment:
Yes, it's used implicitly, MessageBox::addButton(..) returns QPushButton*. Without this include, we get compile error: comparison between distinct pointer types ‘QAbstractButton*’ and ‘QPushButton*’ lacks a cast.